src/ibrowse_http_client.erl

```diff
@@ -465,7 +465,7 @@ accumulate_response(Data, #state{reply_buffer      = RepBuf,
 
 make_tmp_filename(true) ->
     DownloadDir = ibrowse:get_config_value(download_dir, filename:absname("./")),
-    {A,B,C} = now(),
+    {A,B,C} = timestamp(),
```

Comment:
I think it would be `os:timestamp()` not just `timestamp()`

Reply:
I am using this one https://erlang.org/doc/man/erlang.html#timestamp-0 which is the recommended replacement for `now()` according to https://erlang.org/doc/apps/erts/time_correction.html#new-time-api.

Reply:
`erlang:timestamp()` is right but it's not auto-imported I think?

```
1> erlang:timestamp().
{1621,873425,508305}
2> timestamp().
** exception error: undefined shell command timestamp/0
```

Reply:
Interesting, I need to look into what is auto-imported. I figured `erlang` would be imported automatically. Fixing now.